database/seeds/KeyTableSeeder.php

```diff
@@ -0,0 +1,256 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use App\Key;
+
+class KeyTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        Key::create([
+            'key' => 'fantôme',
```

Comment:
I would recommend to not use UTF-8 character in the key.

Reply:
Removed UTF-8 characters in [this commit](https://github.com/antistatique/acapella-text2lpc/pull/19/commits/527e0a03e59a3ec3f5a5c1b92dadc09e9be32043)